pyfr/plugins/integrate.py

```diff
@@ -55,19 +55,32 @@ def __init__(self, intg, cfgsect, suffix=None):
         # Prepare the per element-type info list
         self.eleinfo = []
```

Comment:
self.eleinfo = eleinfo = [] may enable you to avoid the line break in self.eleinfo.append